kubetest/e2e.go

```diff
@@ -37,6 +37,9 @@ func run(deploy deployer, o options) error {
 	os.Setenv("KUBE_CONFIG_FILE", "config-test.sh")
 	// force having batch/v2alpha1 always on for e2e tests
 	os.Setenv("KUBE_RUNTIME_CONFIG", "batch/v2alpha1=true")
+	// force having apps/v1beta2 always on for e2e tests
+	// TODO: remove it once we enable apps/v1beta2 by default
+	os.Setenv("KUBE_RUNTIME_CONFIG", "apps/v1beta2=true")
```

Comment:
I think it should be able to work by implying from this [comment](https://github.com/kubernetes/kubernetes/issues/16117#issue-112872599). :)